seq2simulate/diversity.py

```diff
@@ -92,6 +97,15 @@ def simulate(sequence, working_dir):
                 print "Kept a total of", len(allowed_sequences), "evolved " \
                     "sequences."
                 
+                if hypermutate:
```

Comment:
This is always true and is checking if the package exists.  Means everything always gets hypermutated whether you put --proviral in or not!  You want `hypermutate_seqs`.